README.md

```diff
@@ -1,4 +1,4 @@
-# Sendsmaily Sync for Magento
+# Smaily Email Marketing And Automation
```

Comment:
Changed module name as it is in Magento 2 module. Maybe we should change the repo name also. Something that includes Magento 1 for example `smaily-magento1-extension` and also the comment `Smaily extension for Magento 1.7+`

Reply:
That is a good idea.

But maybe the module should be named `Smaily for Magento 1.x`?

Reply:
Changed to Smaily for Magento 1.x in 608fbd51c1b00c0b7b2dfcd11dda1b068a5c5e16